test/functional/feature_assumeutxo.py

```diff
@@ -152,6 +148,35 @@ def test_invalid_mempool_state(self, dump_output_path):
 
         self.restart_node(2, extra_args=self.extra_args[2])
 
+    def test_snapshot_in_a_divergent_chain(self, dump_output_path):
+        # First rollback node2's chain to the pregenerated one, up to height 199
+        node = self.nodes[2]
```

Comment:
Nit: Because this test involves dealing with multiple nodes, for verbosity and clarity we can call this variable as `second_node`.

Reply:
Thanks for your review. Actually this test uses only one node and I used the same notation used in other tests, but I am willing to change it if others agree.